clog/loggers.py

```diff
@@ -239,6 +242,16 @@ def close(self):
         self.connected = False
 
 
+class MonkLogger(object):
+    """Wrapper around MonkProducer"""
+
+    def __init__(self, client_id, host, port):
+        self.producer = MonkProducer(client_id, host, port)
+
+    def log_line(self, stream, line):
+        self.producer.send_messages(stream, line, None)
```

Comment:
I think send_messages requires a list of messages, so we should have `[line]` instead of `line`.